docs/medical-api/api-reference/patient/bulk-create-patient.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: "Bulk Patient Create"
+description: "Creates multiple Patients in Metriport at once."
+api: "POST /medical/v1/patient/bulk"
+---
+
+The Bulk Patient Create flow allows you to insert up to 100,000 patients at Metriport in a
+single request/job. We'll also populate the medical data for those patients automatically.
+
+This feature avoids the complexity to work within the boundaries of
+[rate limits](/medical-api/more-info/limits#rate-limits) and to coordinate the data
+query for large groups of patients.
```

Comment:
This feature avoids the complexity of working within the boundaries of
[rate limits](/medical-api/more-info/limits#rate-limits) and of coordinating the data
query for large groups of patients.

Reply:
I'd just remove this entirely - it's more to read that doesn't help the user use the feature, it's more like justification for why it exists.

Reply:
done